rspace/src/test/scala/coop/rchain/rspace/ReplayRSpaceTests.scala

```diff
@@ -812,46 +812,49 @@ trait ReplayRSpaceTests
       )
     }
 
-  "replay" should "not allow for ambiguous executions" ignore withTestSpaces { (space, replaySpace) =>
-    val noMatch = Right(None)
-    val empty = space.createCheckpoint()
-    val channel1      = "ch1"
-    val channel2      = "ch2"
-    val key1          = List(channel1, channel2)
-    val patterns: List[Pattern]      = List(Wildcard, Wildcard)
-    val continuation1  = "continuation"
-    val continuation2  = "continuation"
-    val data1         = "datum1"
-    val data2         = "datum2"
-    val data3         = "datum3"
+  "replay" should "not allow for ambiguous executions" in withTestSpaces { (space, replaySpace) =>
+    val noMatch                 = Right(None)
+    val empty                   = space.createCheckpoint()
+    val channel1                = "ch1"
+    val channel2                = "ch2"
+    val key1                    = List(channel1, channel2)
+    val patterns: List[Pattern] = List(Wildcard, Wildcard)
+    val continuation1           = "continuation1"
+    val continuation2           = "continuation2"
+    val data1                   = "datum1"
+    val data2                   = "datum2"
+    val data3                   = "datum3"
 
     //some maliciously 'random' play order
-    space.produce(channel1, data3, false) shouldBe noMatch
-    space.produce(channel1, data3, false) shouldBe noMatch
-    space.produce(channel2, data1, false) shouldBe noMatch
+    space.produce(channel1, data3, false, 0) shouldBe noMatch
+    space.produce(channel1, data3, false, 0) shouldBe noMatch
+    space.produce(channel2, data1, false, 0) shouldBe noMatch
 
-    space.consume(key1, patterns, continuation1, false).right.get should not be empty
+    space.consume(key1, patterns, continuation1, false, 0).right.get should not be empty
     //continuation1 produces data1 on ch2
-    space.produce(channel2, data1, false) shouldBe noMatch
-    space.consume(key1, patterns, continuation2, false).right.get should not be empty
+    space.produce(channel2, data1, false, 1) shouldBe noMatch
+    space.consume(key1, patterns, continuation2, false, 0).right.get should not be empty
     //continuation2 produces data2 on ch2
-    space.produce(channel2, data2, false) shouldBe noMatch
+    space.produce(channel2, data2, false, 2) shouldBe noMatch
```

Comment:
Shouldn't this be `0`?

Reply:
If this is the continuation of a reduction between

```
    space.produce(channel2, data1, false, 1)
    space.consume(key1, patterns, continuation2, false, 0)
```

then max(1,0) + 1 = 2